packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -32,12 +25,13 @@ export type ParticipantListProps = {
   onParticipantRemove?: (userId: string) => void;
 };
 
-const getDefaultRenderer = (
+const defaultRenderer = (
```

Comment:
nit rename to include participants as default renderer doesn't indicate what it is rendering

```suggestion
const renderParticipants = (
```
or
```suggestion
const onRenderParticipantsDefault = (
```